packages/api-server/api_server/repositories/sql.py

```diff
@@ -0,0 +1,33 @@
+import logging
```

Comment:
I'm not very certain on this, for one, the model themselves already expose the query functions, I guess we still need this to translate between ros messages <-> sql models, and I guess it's also nice to consolidate the queries here.

`sql.py` is just short for `sql_repository.py` because I feel that it is redundant to repeat the repo
`from .repositories.sql_repository import SqlRepository`
vs
`from .repositories.sql import SqlRepository`

Reply:
Oh, ok, that makes sense. Thanks